src/core/File.ts

```diff
@@ -298,11 +298,11 @@ export class File {
         this.isLoaded = true;
     }
 
-    public makeAnalysis(parserOptions?: any) {
+    public makeAnalysis(parserOptions?: any, traversalOptions?: { plugins: any[] }) {
```

Comment:
Are you suggesting I create a "`TraversalPlugin`" interface? (i don't see one currently)
They're just typed as any right now..
https://github.com/calebboyd/fuse-box/blob/master/src/analysis/FileAnalysis.ts#L14

```typescript
interface TraversalPlugin {
    onNode(file: File, node: any, parent: any): void
    onEnd(file: File): void
}
```